.github/workflows/scala-ci.yml

```
@@ -30,9 +30,9 @@ jobs:
           ~/.ivy2/cache
         key: sbt-ivy-cache-spark-${{ matrix.spark-version}}-java-${{ matrix.java-version }}
     - name: Check scalafmt
-      run: build/sbt -Dspark.version=${{ matrix.spark-version }} root/scalafmtCheckAll connect/scalafmtCheckAll
+      run: build/sbt -Dspark.version=${{ matrix.spark-version }} core/scalafmtCheckAll connect/scalafmtCheckAll
```

All of these can just be a single call now
```suggestion
      run: build/sbt -Dspark.version=${{ matrix.spark-version }} scalafmtCheckAll
```